src/JitInterface/src/CorInfoImpl.cs

```diff
@@ -1700,17 +1701,30 @@ private bool isFieldStatic(CORINFO_FIELD_STRUCT_* fldHnd)
         {
             return HandleToObject(fldHnd).IsStatic;
         }
-
+        private enum ICorDebugInfo
```

Comment:
This enum should be in JitInterface/src/CorInfoTypes.cs. The other ICorDebugInfo structures translated from C++ to C# are there as well.